tests/css-pseudo-4.js

```diff
@@ -87,7 +87,20 @@ export default {
 			links: {
 				dev: '#details-content-pseudo',
 			},
-			tests: ['::details-content'],
+			tests: [
+				'::details-content',
+				'::details-content::first-letter',
```

Comment:
`::first-letter` can also have `::prefix` and `::postfix` as subsequent pseudo-elements. So maybe we should test for them here, too.